src/main/java/Problem3/BookFiction.java

```diff
@@ -9,12 +9,18 @@
     public BookFiction(String title, String author, String genres) {
         // homework
         // tip: use the 'super' keyword
+        super(title, author);
+        this.genres = genres;
     }
 
     // copy constructor
     public BookFiction(BookFiction anotherBook) {
         // homework
         // tip: use the 'super' keyword
+        super(anotherBook.title, anotherBook.author);
```

Comment:
super class Book already has a copy construction. why not just use that and save the id copy on line 23?